src/ec/mod.rs

```diff
@@ -47,6 +47,40 @@ pub const EDWARDS448: &str = "edwards448";
 pub const CURVE25519: &str = "curve25519";
 pub const CURVE448: &str = "curve448";
 
+fn bits_to_bytes(bits: usize) -> usize {
+    (bits + 7) / 8
+}
+
+fn ec_ws_point_size(bits: usize) -> usize {
```

Comment:
in the function name, its not clear to me what the `ws` means on the first read so I would propose to either choose a better name or do explaining comment.

Reply:
changed ws ->secp